ingeniamc/k2-base-camp
# DR0053A-586 Add drive status LEDs

### Docs

https://novantamotion.atlassian.net/browse/DR0053A-586

### Test

1. Connect to a drive
2. Verify that there are now status LEDs at the top of the screen.
3. Enable / disable the motors and verify that the corresponding status LED gets updated accordingly.

![image](https://github.com/ingeniamc/k2-base-camp/assets/64021569/d45db547-83d4-471d-8d2c-ee489f096a38)
